javascript/javascript1/week3/Smart_ease.js

```diff
@@ -0,0 +1,61 @@
+const notes = [];
+
+function saveNote(content, id) {
+    let writeNotes = { content: content, id: id }
+    notes.push(writeNotes);
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("cleaning", 3);
+
+console.log(notes);
+
+function getNote(id) {
+    for (let i = 0; i < notes.length; i++) {
+        if (id === notes[i].id) {
+            return notes[i];
+          
+        };
+        
+    }
+
+    return `This id=${id} is not valid.`;
+}
+  const firstNote = getNote(4);
+  console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+  function logOutNotesFormatted() {
+    for (let i = 0; i < notes.length; i++) {
+        console.log(`The note with id:${notes[i].id}, has the following note text: ${notes[i].content}`);
+    }
+};
+
+logOutNotesFormatted();
+
+//Smart phone usage app
+
+const activities = [];
+function addActivity(date, activity, duration) {
+
+    activities.push({ date: date, activity: activity, duration: duration })
+
+}
+addActivity("23/7-18", "Youtube", 30);
```

Comment:
this is not a valid date. Please check `Date` in javascript to see how you can write valid time data